mParticle-ComScore.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
     s.ios.deployment_target = "8.0"
     s.ios.source_files      = 'mParticle-ComScore/*.{h,m,mm}'
     s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 6.12.0'
```

Comment:
Yeah - I've been thinking we should pin all of the versions in all the kits to always match. Perhaps we should do that separately from this PR as part of a larger restructure effort, though?